gbrl/learners/actor_critic_learner.py

```
@@ -188,6 +191,82 @@ def predict_critic(self, obs: NumericalData,
                                       tensor)
         return pred_values
 
+    def compress(self, trees_to_keep: int, gradient_steps: int, features: NumericalData,
+                 actions: Optional[th.Tensor] = None, log_std: Optional[th.Tensor] = None,
+                 method: str = 'first_k', dist_type: str = 'deterministic',
+                 optimizer_kwargs: Optional[Dict[str, Any]] = None,
+                 temperature: float = 1.0, lambda_reg: float = 1.0, **kwargs):
```

Ruff (0.14.14)

[warning] Missing type annotation for `**kwargs`

(ANN003)